rskj-core/src/main/java/co/rsk/net/handler/txvalidator/TxValidatorIntrinsicGasLimitValidator.java

```diff
@@ -38,8 +39,8 @@ public TxValidatorIntrinsicGasLimitValidator(RskSystemProperties config) {
 
     @Override
     public boolean validate(Transaction tx, AccountState state, BigInteger gasLimit, BigInteger minimumGasPrice, long bestBlockNumber, boolean isFreeTx) {
-        BlockHeader blockHeader = new BlockHeader(new byte[]{},
-                new byte[]{},
+        BlockHeader blockHeader = new BlockHeader(new Keccak256(new byte[]{}),
```

Comment:
There's a zeroHash and then there's this construction with an empty array. It sounds weird.